jobs/cloud_controller_ng/spec

```diff
@@ -85,8 +85,8 @@ provides:
   - ccdb.read_timeout
   - ccdb.ssl_verify_hostname
   - ccdb.ca_cert
-- name: directories_to_backup
```

Comment:
Does outright deleting a link like this cause upgrade issues if someone else is consuming it?

Reply:
Hi @tcdowney, this could be a breaking change for someone. We could not find any consumers of the `directories_to_backup` link in any cf-deployment or Pivotal BOSH releases. Changing this link name is a major version change. Let's avoid that by keeping it the same.